mss/Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM hudl/ffmpeg
```

Comment:
I have a couple thoughts on this one - we could keep it as an `opencv` image, but tag it with `3.1`. We might need to update our build steps to handle it, and it could also mean switching up our directory structure (`image_name/tag/Dockerfile` or something?).

Alternatively, since this evolved into being used for a specific project and has dependencies that aren't strictly OpenCV related, should we just move this into our `docker-private` repo instead?